core/src/main/scala/com/banno/kafka/consumer/ConsumerOps.scala

```diff
@@ -19,9 +19,11 @@ package consumer
 
 import cats.*
 import cats.effect.*
+import cats.effect.implicits.*
```

Comment:
Minor nit: `import cats.effect.syntax.all.*` is the idiomatic way to get to the implicits instances.